commons/src/main/java/org/acme/common/HttpClientConfiguration.java

```diff
@@ -120,39 +105,41 @@ public static ManagedHttpClient newManagedHttpClient() {
+            HttpRoutePlanner routePlanner = new DefaultProxyRoutePlanner(new HttpHost(proxyInfo.getHost(), proxyInfo.getPort())) {
                 @Override
                 public HttpRoute determineRoute(
                         final HttpHost host,
                         final HttpRequest request,
                         final HttpContext context) throws HttpException {
-                    if (isProxy(proxyInfo.noProxy, host)) {
+                    if (isProxy(proxyInfo.getNoProxy(), host)) {
                         return super.determineRoute(host, request, context);
                     }
                     return new HttpRoute(host);
                 }
             };
             clientBuilder.setRoutePlanner(routePlanner);
-            if (StringUtils.isNotBlank(proxyInfo.username) && StringUtils.isNotBlank(proxyInfo.password)) {
-                if (proxyInfo.domain != null) {
-                    credsProvider.setCredentials(AuthScope.ANY, new NTCredentials(proxyInfo.username, proxyInfo.password, proxyInfo.domain, null));
+            if (StringUtils.isNotBlank(proxyInfo.getUsername()) && StringUtils.isNotBlank(proxyInfo.getPassword())) {
+                if (proxyInfo.getDomain() != null) {
+                    credsProvider.setCredentials(AuthScope.ANY, new NTCredentials(proxyInfo.getUsername(), proxyInfo.getPassword(), proxyInfo.getDomain(), null));
                 } else {
-                    credsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(proxyInfo.username, proxyInfo.password));
+                    credsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(proxyInfo.getUsername(), proxyInfo.getPassword()));
                 }
             }
         }
         // When a proxy is enabled, turn off certificate chain of trust validation and hostname verification
-        if (proxyInfo != null && proxyInfo.noProxy == null) {
+        if (proxyInfo != null && proxyInfo.getNoProxy() == null) {
             try {
                 final SSLContext sslContext = SSLContextBuilder
                         .create()
                         .loadTrustMaterial(new TrustAllStrategy())
                         .build();
-                final Registry<ConnectionSocketFactory> registry = RegistryBuilder.<ConnectionSocketFactory> create()
+                final Registry<ConnectionSocketFactory> registry = RegistryBuilder.<ConnectionSocketFactory>create()
                         .register("http", PlainConnectionSocketFactory.INSTANCE)
                         .register("https", new SSLConnectionSocketFactory(sslContext, NoopHostnameVerifier.INSTANCE))
                         .build();
                 connectionManager = new PoolingHttpClientConnectionManager(registry);
+                new PoolingHttpClientConnectionManagerMetricsBinder(connectionManager, "metaAnalyzer").bindTo(meterRegistry);
```

Comment:
This class is in `commons`, but the name in the metrics will say `metaAnalyzer`. I haven't tested it, but perhaps you can inject the `quarkus.application.name` property here, and use that instead? https://quarkus.io/guides/all-config#quarkus-core_quarkus.application.name

That way the client should always use the name of the application it's used in.